client/my-sites/stats/activity-log-date/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import Button from 'components/button';
+import ActivityLogItem from '../activity-log-item';
+
+class ActivityLogDate extends Component {
+	constructor( props, context ) {
+		super( props, context );
+
+		this.state = {
+			timestamp: this.props.logs[ 0 ].timestamp
```

Comment:
ah yep, [not yet](https://github.com/Automattic/wp-calypso/pull/10862/files#diff-8d4a72af822b57ca42cdc8857fbf5ccdR19)... 

Will remove for the sake of this PR